pkg/apiserver/api_installer.go

```diff
@@ -286,6 +290,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	scope := mapping.Scope
 	nameParam := ws.PathParameter("name", "name of the "+kind).DataType("string")
 	pathParam := ws.PathParameter("path", "path to the resource").DataType("string")
+	exportParam := ws.QueryParameter("export", "strip fields for export").DataType("boolean")
+	exactParam := ws.QueryParameter("exact", "should export be exact").DataType("boolean")
```

Comment:
done.  @deads2k I'm not sure you're right about --exact, it seems to mean "strip cluster-specific fields like namespace"